### ISSUE TITLE
Method incorrectly reported as removed

### ISSUE THREAD

**Issue by bric3** [lvc/japi-compliance-checker#34]
I work on the mockito project and noticed a release that has backward compatibility issue ([report on ABI-Laboratory](https://abi-laboratory.pro/java/tracker/compat_report/mockito/2.2.7/2.2.8/8eb4b/bin_compat_report.html)), while the only changes were :

```diff
- Answer1<T,A>.answer(A p1)
+ Answer1<T,A0>.answer(A0 p1) throws Throwable 
``` 

- renamed the type parameters
- added throws declaration

These methods should not be reported as removed, as the type parameter renaming is binary compatible, and the throws declaration is compiler visible only but it is backward compatible.

**Comment by lvc** [comment]
Hi,

This is fixed by commit 9d78fb028752b905326eb67f9748ff1646102746.

Thank you.

**Comment by bric3** [comment]
No thanks to you @lvc ;)
